docs/_includes/include-lines-tags.adoc

```diff
@@ -88,9 +89,37 @@ snippet b
 
 Notice that none of the lines with the tag directives are displayed.
 
-NOTE: for XML files you can use the `+<!-- tag::name[] -->+` and `+<!-- end::name[] -->+` delimiters
+==== By expressions using tags
```

Comment:
I think we should name this section "Tag filtering". We are essentially getting more fine-grained about which tagged regions should be selected.